.docker/experimental/Dockerfile

```diff
@@ -1,28 +0,0 @@
-# moveit/moveit:melodic-experimental
```

Comment:
Let's just keep this?

      TODO: convert to ROS2 similar to https://github.com/ros-planning/moveit2/pull/46

Reply:
For moveit2 I would like to be more selective about what we remove and what we keep. If we restore the benchmarking code later we can always bring this file back.

Reply:
this technically supports warehouse_ros and the warehouse functionality. but i do not have a strong opinion about keeping it or not, so ok.